pkg/blobstore/blob_access_mutable_proto_store.go

```diff
@@ -75,7 +75,8 @@ type blobAccessMutableProtoStore[T any, TProto interface {
 func NewBlobAccessMutableProtoStore[T any, TProto interface {
 	*T
 	proto.Message
-}](initialSizeClassCache blobstore.BlobAccess, maximumMessageSizeBytes int) MutableProtoStore[TProto] {
+}](initialSizeClassCache blobstore.BlobAccess, maximumMessageSizeBytes int,
```

Comment:
What's going on here, by the way? Looks like an odd way to format this code.

Reply:
Maybe we should leave gofumpt pinned at 0.2.1?

Reply:
bb-storage and the others have 0.3.0. Seems like I have to run gofumpt twice to get it stable, splitting the line the second time.

Reply:
I'd rather stick to 0.2.1 in this repo if that prevents the unnecessary formatting.

Reply:
OK, fixing...

Reply:
Went back to gofumpt 0.2.1 and added a comment. Also checked 0.4.0 but it formats the same way as 0.3.0.